baremaps-server-ogcapi/src/main/resources/schemas/collection.yaml

```diff
@@ -4,41 +4,45 @@ required:
   - links
```

Comment:
Out of curiosity, have these changes been made upstream in the OGC API repository?

Reply:
The properties after the comment are specific to the studio, the rest is from the ogc api repository. Since the model schema does not prohibit additional properties, this is valid.

Reply:
Ok, thank you for the clarification